hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JetService.java

```diff
@@ -243,8 +244,12 @@ public ClassLoader getClassLoader(long jobId) {
     }
 
     @Override
-    public void handle(Packet packet) throws IOException {
-        networking.handle(packet);
+    public void accept(Packet packet) {
+        try {
```

Comment:
we could use `uncheckedRun()`

Reply:
It's quite a hot path here, `uncheckRun` will create one throwaway lambda for each packet, I wouldn't do it.